cts/cli/regression.cibadmin.exp

```diff
@@ -37,7 +37,7 @@
 =#=#=#= End test: Validate CIB - OK (0) =#=#=#=
 * Passed: cibadmin              - Validate CIB
 =#=#=#= Begin test: Digest calculation =#=#=#=
-Digest: 01560bbfb91f23e402b8ce399b2e7e32
+Digest: 5bfd0f70973217c2a4735960dfafbb55
```

Comment:
In previous cts-cli, we used output redirection in the test to get rid of the actual digest, leaving just "Digest:" for the output.  I decided to drop that regex so we'd get real output here.  If it turns out to be a pain, we can definitely add the redirection back in.